packages/admin-panel/src/icons/ExportIcon.jsx

```diff
@@ -0,0 +1,28 @@
+/*
+ * Tupaia
+ * Copyright (c) 2017 - 2024 Beyond Essential Systems Pty Ltd
+ */
+import React from 'react';
+import { SvgIcon } from '@material-ui/core';
+
+export const ExportIcon = props => {
+  return (
+    <SvgIcon
+      {...props}
+      width="17"
+      height="17"
+      viewBox="0 0 17 17"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
```

Comment:
Just so we at least have the choice to overriding the width and height with `<ExportIcon width={} height={} />`

```suggestion
      width="17"
      height="17"
      viewBox="0 0 17 17"
      fill="none"
      xmlns="http://www.w3.org/2000/svg"
      {...props}
```

Ditto `ImportIcon.jsx`